src/frontend/js/settings/settings.dev.dist.ts

```diff
@@ -0,0 +1,25 @@
+// This configuration file can be overridden with settings.dev.ts file.
+// `$ cp settings.local.dist.ts settings.local.ts`, then change what's needed
+// in local environment in settings.local.ts.
+import { DevDemoUser } from 'api/lms/dummy';
+
+export const REACT_QUERY_SETTINGS = {
+  // disable react query cache
+  staleTimes: {
+    session: 0,
+    sessionItems: 0,
+  },
+};
```

Comment:
Not a good idea I think to disable react query cache by default. At least session.